bugbot/rules/variant_expiration.py

```diff
@@ -303,6 +303,12 @@ def handle_bug(self, bug, data):
                 "comment": {
                     "body": f"The variant has been removed from the [variants.yml]({VARIANTS_SEARCHFOX_URL}) file."
                 },
+                "flags": [
+                    {
+                        "name": "needinfo",
+                        "status": "X",
+                    }
+                ],
```

Comment:
We need to specify which needinfo should be cleared. The BMO docs could be helpful: https://bmo.readthedocs.io/en/latest/api/core/v1/bug.html#update-bug